azurelinuxagent/common/protocol/extensions_goal_state.py

```diff
@@ -82,7 +81,7 @@ def agent_manifests(self):
         raise NotImplementedError()
 
     @property
-    def ext_handlers(self):
+    def extensions(self):
```

Comment:
Renamed ExtHandler to Extension, which a more appropriate term.

Reply:
In the new terminology an extension is an Extension and the settings are ExtensionSettings. Actually, I wanted to use ExtensionGoalState for the former, but today we use ExtHandler both for the extension goal state and, for example, extensions installed on disk, so that requires more refactoring.

As far as the additinal renaming, I agree, see my overall comment on renames.